redwood-layout-schema/src/main/kotlin/app/cash/redwood/layout/widgets.kt

```diff
@@ -169,3 +169,77 @@ public data class Spacer(
   @Default("Dp(0.0)")
   val height: Dp,
 )
+
+/**
+ * Lays out widgets along the z-axis in a column.
+ *
+ * Minimum and maximum heights do not include margins.
+ */
+@Widget(4)
+public data class Box(
+  /**
+   * Sets whether the box's width will match its widest child ([Constraint.Wrap]) or match the width
+   * of its parent ([Constraint.Fill]).
+   */
+  @Property(1)
+  @Default("Constraint.Wrap")
+  val width: Constraint,
+
+  @Property(2)
+  @Default("Dp(0.0)")
+  val minWidth: Dp,
+
+  @Property(3)
+  @Default("Dp(Double.MAX_VALUE)")
+  val maxWidth: Dp,
+
+  /**
+   * Sets whether the box's height will match its tallest child ([Constraint.Wrap]) or match the
+   * height of its parent ([Constraint.Fill]).
+   */
+  @Property(4)
+  @Default("Constraint.Wrap")
+  val height: Constraint,
+
+  @Property(5)
+  @Default("Dp(0.0)")
+  val minHeight: Dp,
```

Comment:
Do folks think that these would be useful?

Reply:
I think it's fine to be here, as long as it is a no-op when not assigned

Reply:
cc: @swankjesse going to remove these